src/phasorpy/phasor.py

```diff
@@ -444,18 +450,87 @@ def phasor_calibrate(
     (array([-1.56, 1.934, 0.3279]), array([5.985, 10.6, 1.986]))
```

Comment:
I would prefer to have a more realistic example, where the calibrated coordinates end up in the range 0..1.